scripts/local_multi_setup.sh

```diff
@@ -9,132 +13,106 @@ rm -rf $HOME/.sedad/
-sed -i -E 's|prometheus = false|prometheus = true|g' $VALIDATOR3_CONFIG
-sed -i -E 's|prometheus_listen_addr = ":26660"|prometheus_listen_addr = ":26620"|g' $VALIDATOR3_CONFIG
-
-# validator4
-sed -i -E 's|tcp://127.0.0.1:26658|tcp://127.0.0.1:26649|g' $VALIDATOR4_CONFIG
-sed -i -E 's|tcp://127.0.0.1:26657|tcp://127.0.0.1:26648|g' $VALIDATOR4_CONFIG
-sed -i -E 's|tcp://0.0.0.0:26656|tcp://0.0.0.0:26647|g' $VALIDATOR4_CONFIG
-sed -i -E 's|allow_duplicate_ip = false|allow_duplicate_ip = true|g' $VALIDATOR4_CONFIG
-sed -i -E 's|prometheus = false|prometheus = true|g' $VALIDATOR4_CONFIG
-sed -i -E 's|prometheus_listen_addr = ":26660"|prometheus_listen_addr = ":26610"|g' $VALIDATOR4_CONFIG
-
-
-# copy validator1 genesis file to validator2-4
-cp $HOME/.sedad/validator1/config/genesis.json $HOME/.sedad/validator2/config/genesis.json
-cp $HOME/.sedad/validator1/config/genesis.json $HOME/.sedad/validator3/config/genesis.json
-cp $HOME/.sedad/validator1/config/genesis.json $HOME/.sedad/validator4/config/genesis.json
+sed -i '' -E 's|tcp://127.0.0.1:26658|tcp://127.0.0.1:26655|g' $VALIDATOR2_CONFIG
+# sed -i '' -E 's|tcp://127.0.0.1:26657|tcp://127.0.0.1:26654|g' $VALIDATOR2_CONFIG
+sed -i '' -E 's|tcp://0.0.0.0:26657|tcp://127.0.0.1:26654|g' $VALIDATOR2_CONFIG
+sed -i '' -E 's|tcp://0.0.0.0:26656|tcp://0.0.0.0:26653|g' $VALIDATOR2_CONFIG
+sed -i '' -E 's|allow_duplicate_ip = false|allow_duplicate_ip = true|g' $VALIDATOR2_CONFIG
+sed -i '' -E 's|prometheus = false|prometheus = true|g' $VALIDATOR2_CONFIG
+sed -i '' -E 's|prometheus_listen_addr = ":26660"|prometheus_listen_addr = ":26630"|g' $VALIDATOR2_CONFIG
+
+# modify validator1 genesis file to validator2
+jq '.consensus.params.abci.vote_extensions_enable_height = "15"' $HOME/.sedad/validator1/config/genesis.json > temp.json && mv temp.json $HOME/.sedad/validator1/config/genesis.json
 
+cp $HOME/.sedad/validator1/config/genesis.json $HOME/.sedad/validator2/config/genesis.json
 
 # copy tendermint node id of validator1 to persistent peers of validator2-4
-sed -i -E "s|persistent_peers = \"\"|persistent_peers = \"$(sedad tendermint show-node-id --home=$HOME/.sedad/validator1)@localhost:26656\"|g" $HOME/.sedad/validator2/config/config.toml
-sed -i -E "s|persistent_peers = \"\"|persistent_peers = \"$(sedad tendermint show-node-id --home=$HOME/.sedad/validator1)@localhost:26656\"|g" $HOME/.sedad/validator3/config/config.toml
-sed -i -E "s|persistent_peers = \"\"|persistent_peers = \"$(sedad tendermint show-node-id --home=$HOME/.sedad/validator1)@localhost:26656\"|g" $HOME/.sedad/validator4/config/config.toml
+sed -i '' -E "s|persistent_peers = \"\"|persistent_peers = \"$($BIN tendermint show-node-id --home=$HOME/.sedad/validator1)@localhost:26656\"|g" $HOME/.sedad/validator2/config/config.toml
 
 # start all four validators
 tmux new-session -s validator1 -d 
-tmux send -t validator1 'sedad start --home=$HOME/.sedad/validator1' ENTER
+tmux send -t validator1 'BIN=./build/sedad' ENTER
```

Comment:
Should be listed as a dev dep for testing IMO. Also would be nice if the script errors if tmux, or any other program needed, isn't installed at the top of the file.